experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -404,18 +432,23 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
 
       const headers = request.headers;
 
+      const spanAttributes = utils.getIncomingRequestAttributes(request, {
+        component: component,
+        serverName: instrumentation._getConfig().serverName,
+        hookAttributes: instrumentation._callStartSpanHook(
+          request,
+          instrumentation._getConfig().startIncomingSpanHook
+        ),
+      });
+
       const spanOptions: SpanOptions = {
         kind: SpanKind.SERVER,
-        attributes: utils.getIncomingRequestAttributes(request, {
-          component: component,
-          serverName: instrumentation._getConfig().serverName,
-          hookAttributes: instrumentation._callStartSpanHook(
-            request,
-            instrumentation._getConfig().startIncomingSpanHook
-          ),
-        }),
+        attributes: spanAttributes,
       };
 
+      const startTime = hrTime();
+      let metricAttributes: MetricAttributes = utils.getIncomingRequestMetricAttributes(spanAttributes, { component: component });
```

Comment:
`component` should already exist on `spanAttributes`, right?

Reply:
HTTP_SCHEME was not there, so I'm adding as part of this PR as well, it is required according to the spec.
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#http-server-semantic-conventions